pybossa/view/account.py

```diff
@@ -1228,16 +1237,20 @@ def taskbrowse_bookmarks(user_name, short_name):
     if current_user.name != user_name:
         return abort(404)
 
+    order_by  = request.args.get('order_by', None, type=str)
+    desc_str  = request.args.get('desc', None)
+    desc = True if desc_str == 'true' else False if desc_str == 'false' else None
```

Comment:
How about the following equivalent way?
```
desc = {'true': True, 'false': False}.get(desc_str)
```
You might want to convert desc_str to lower case as well, in that case, you will need ```desc_str.lower()```

Reply:
`desc_str.lower()` fails and throws an error when `desc_str=None`